### ISSUE TITLE
DescriptionList - lists should have unique aria-labels

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7277]
**Describe the enhancement or change**
In pages where multiple DescriptionLists are rendered, each should have a unique aria-label to follow the recommended approach for AT that show each list on a page i.e. in VoiceOver's rotor menu. When there's more than 1 on a page, you see a list of just "definition list" which makes it difficult to traverse via that menu.

**Any other information?**
